src/test/java/org/broadinstitute/consent/http/WithMockServer.java

```diff
@@ -23,6 +23,24 @@ public interface WithMockServer {
   DockerImageName IMAGE =
       DockerImageName.parse("mockserver/mockserver:mockserver-" + getMockServerVersion());
 
+  static String getMockServerVersion() {
+    String version = "5.15.0";
```

Comment:
This version was being set at 5.12.0, but the testcontainer was pulling in v.5.15.0.

Reply:
Really nice catch here. My approach for swagger is to pull the version dynamically from `mvn.properties`. In my local value, I see:
```
...
mockserver.version=5.15.0
...
```
See https://github.com/DataBiosphere/consent/blob/f9dae18c771324472613b22cf3b0e7873f65a6f7/src/main/java/org/broadinstitute/consent/http/resources/SwaggerResource.java#L38 for how we're doing that.

Reply:
Your point is well taken, @pshapiro4broad.  If we decide on that, maybe we make another ticket for this change - since it's beyond getting the tests to pass locally.  

The reason the vast majority of the code below is green is because of formatting changes that alphabetized the order of methods.